Adyen/Util/HMACValidator.cs

```diff
@@ -85,6 +85,20 @@ public bool IsValidHmac(NotificationRequestItem notificationRequestItem, string
             var merchantSign = notificationRequestItem.AdditionalData[Constants.AdditionalData.HmacSignature];
             return string.Equals(expectedSign, merchantSign);
         }
+
+        public bool isValidHmac(string hmacKey, string hmacSignature, string payload)

```

Comment:
Actually the validation is done the same way, except for the old webhook payload gets formatted a bit differently. I do agree with the similar naming being confusing though. I'll change the name instead of overloading like this.